ansible_collections/arista/avd/roles/eos_designs/templates/network_services/router-bgp-vrfs.j2

```diff
@@ -24,84 +24,94 @@
 {%                 endif %}
 {%             endfor %}
     {{ vrf.name }}:
+{%             if vrf.name == 'default' %}
+{%                 if default_vrf.svi_subnets %}
+{%                     do route_targets.export.evpn.append('route-map RM-EVPN-OVERLAY-PEERS-DEFAULT-VRF-OUT') %}
```

Comment:
Since this RM is applied on evpn export IMO it should be named something like `RM-EVPN-EXPORT-VRF-DEFAULT`